pilot/pkg/bootstrap/server.go

```diff
@@ -574,6 +574,14 @@ func (s *Server) initKubeClient(args *PilotArgs) error {
 		if err != nil {
 			return fmt.Errorf("failed creating kube client: %v", err)
 		}
+
+		memberRollName := args.RegistryOptions.KubeOptions.MemberRollName
+		if memberRollName != "" {
+			err := s.kubeClient.AddMemberRoll(args.Namespace, memberRollName)
+			if err != nil {
+				return fmt.Errorf("failed creating member roll: %v", err)
+			}
```

Comment:
Alternatively you can consider this:
```suggestion
			if err := s.kubeClient.AddMemberRoll(args.Namespace, memberRollName); err != nil {
				return fmt.Errorf("failed creating member roll: %v", err)
			}
```

Reply:
Thanks, I prefer this notation. I just modified only not working code, but I will change it as you suggest.

Reply:
Done